### PR TITLE
Main

### PR DESCRIPTION
**Pull request by wahebtalal**
## Documentation
- [ ] My pull request requires an update on the documentation repository.

## Branch Selection

- [x] Target Branch: master

### CODE REVIEW

File: packages/Webkul/Shop/src/Http/Controllers/SearchController.php [hunk 1]
```diff
@@ -25,7 +25,7 @@ public function __construct(
     public function index()
     {
         $this->validate(request(), [
-            'query' => ['required', 'string', 'regex:/^[^\\\\]+$/u'],
+            'query' => ['string', 'regex:/^[^\\\\]+$/u'],
```
Comment (amit-webkul) [inline packages/Webkul/Shop/src/Http/Controllers/SearchController.php:28]: Here required should be added.

**Review by amit-webkul — CHANGES_REQUESTED** [review]

**Comment by amit-webkul** [comment]
@wahebtalal 

Thank you for your contribution to Bagisto.

I have reviewed your pull request and added some comments. Please review these comments and update the pull request accordingly. If you have any concerns or questions about the comments, feel free to respond directly to them.

We appreciate your efforts.

**Comment by amit-webkul** [comment]
Any Updates ?

**Comment by devansh-webkul** [comment]
Feel free to raise a new PR.

**Pull request closed without merging**